SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -435,7 +432,10 @@ void USpatialNetDriver::CreateAndInitializeLoadBalancingClasses()
 
 		if (WorldSettings == nullptr || WorldSettings->DefaultLayerLockingPolicy == nullptr)
 		{
-			UE_LOG(LogSpatialOSNetDriver, Error, TEXT("If EnableMultiWorker is set, there must be a Locking Policy set. Using default policy."));
+			if (WorldSettings->DefaultLayerLoadBalanceStrategy != nullptr)
+			{
+				UE_LOG(LogSpatialOSNetDriver, Error, TEXT("If a Load balancing strategy is set, there must be a Locking Policy set. Using default policy."));
```

Comment:
this checks load balancing strat but log refers to locking policy?

Reply:
The locking policy is checked on 433. It's now valid to have neither LBStrategy nor Locking Policy set, but if you have a LB, you should also have a locking policy.

Reply:
ah yea, my bad

Reply:
Although I do need a WorldSettings != nullptr check here.

Reply:
The WorldSEttings==nullptr conditional on the line above seems weird here, if that's true, we still immediately check WorldSettings->DefaultLayerLoadBalanceStrategy - should we have a separate early exit for that being nullptr?

Reply:
Actually, I refactored this to make the logic simpler and clearer by using the bEnableMultiWorkerflag.